lib/run_loop/device_agent/client.rb

```diff
@@ -38,7 +38,7 @@ class HTTPError < RuntimeError; end
 
         # Ignored in the XTC.
         # This key is subject to removal or changes
-        :device_agent_install_timeout => RunLoop::Environment.ci? ? 240 : 20,
+        :device_agent_install_timeout => RunLoop::Environment.ci? ? 240 : 120,
```

Comment:
Fixed this problem here: https://github.com/calabash/run_loop/pull/765/commits/ed61a41ff4ceb3789695e7c29e59153e7e457afe took advantage to merge your PR and releases a new gem version since some people complained about DeviceAgent not being up-to-date with master